### PR TITLE
[DOC-795] Give advice about query performance in a mixed-version cluster

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-795] Give advice about query performance in a mixed-version cluster

## Previews
[src/current/cockroachcloud/upgrade-to-v22.1.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v22.1.html)
[src/current/cockroachcloud/upgrade-to-v22.2.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v22.2.html)
[src/current/cockroachcloud/upgrade-to-v23.1.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v23.1.html)
[src/current/v22.1/upgrade-cockroach-version.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/v22.1/upgrade-cockroach-version.html)
[src/current/v22.2/upgrade-cockroach-version.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/v22.2/upgrade-cockroach-version.html)
[src/current/v23.1/upgrade-cockroach-version.md](https://deploy-preview-17318--cockroachdb-docs.netlify.app/docs/v23.1/upgrade-cockroach-version.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by RaduBerinde — APPROVED** [review]
Looks good, thanks!

File: src/current/cockroachcloud/upgrade-to-v22.1.md [hunk 1]
```diff
@@ -27,8 +27,11 @@ The upgrade process depends on the number of nodes in your cluster. Select wheth
...
+In a multi-node cluster, the upgrade does not interrupt the cluster's overall health and availability. An admin stops one node at a time and restarts it with the new version, waits a few minutes to observe the upgraded node's behavior, then moves on to the next node. This "rolling upgrade" takes approximately 4-5 minutes per node and is enabled by CockroachDB's [multi-active availability](../{{site.current_cloud_version}}/multi-active-availability.html) design.
+
+{{site.data.alerts.callout_danger}}
+After beginning a major-version upgrade, Cockroach Labs recommends upgrading all nodes as quickly as possible. In a cluster with nodes running different major versions of CockroachDB, a query that is sent to an upgraded node can be distributed only among other upgraded nodes. Data accesses that would otherwise be local may become remote, and the performance of these queries can suffer.
```
Comment (ianjevans) [inline src/current/cockroachcloud/upgrade-to-v22.1.md:33]: I don't think the user really has control over how quickly the nodes are upgraded in Dedicated. You basically trigger an upgrade and just observe the progress. So I guess I'm wondering what the user is expected to do w/r/t "upgrading all nodes as quickly as possible."

**Review by ianjevans — APPROVED** [review]
Left a context nit, but it definitely shouldn't get in the way of merging.

**Comment by mdlinville** [comment]
> Left a context nit, but it definitely shouldn't get in the way of merging.

How silly of me. I edited the Cloud topics instead of the self-hosted topics like I intended. Let me fix it.

File: src/current/cockroachcloud/upgrade-to-v22.1.md (hunk 1 above)
Comment (mdlinville) [reply src/current/cockroachcloud/upgrade-to-v22.1.md]: I reverted adding the note but changed "An admin" to "Cockroach Cloud" for these pages.

**Review by ianjevans — APPROVED** [review]

**Pull request merged**